azure-advanced/Packer/ansible-apim/CONTRIBUTING.md

```diff
@@ -0,0 +1,41 @@
+# Contributing to ansible-apim
```

Comment:
@rasanjaya85 let's remove this which is specific to Ansible resource repository and add a contributors' guide specific to this repository.